front/app/components/admin/SubmitWrapper/index.tsx

```diff
@@ -1,4 +1,4 @@
-import React, { PureComponent, FormEvent } from 'react';
+import React, { FormEvent, useRef, useEffect } from 'react';
```

Comment:
Refactored this to a functional component because it was horrible and causing tons of console warnings